### PR TITLE
Update to use pg 15

### PR DESCRIPTION
**Pull request by c-simpson**
We'd seen weird connection drops with 13.2.  15.4 seems to fix it.

### CODE REVIEW

**Comment by carrollsa** [comment]
Do we also need to update the tag value to `"15.4-alpine"` in `helm-chart/values.yaml`?

File: src/docker-compose.yml [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+
   db:
     container_name: paws-compose-db
     ports:
       - "5432:5432"
-    image: postgres:13.2-alpine
+    image: postgres:15.4-alpine
     volumes:
-      - postgres:/var/lib/postgresql/data
+      - postgres15:/var/lib/postgresql15/data
```
Comment (carrollsa) [inline src/docker-compose.yml:11]: Just for my own learning, can you tell me why we need to update volume name and directory here?

**Comment by c-simpson** [comment]
> Do we also need to update the tag value to "15.4-alpine" in helm-chart/values.yaml?

Yes, good catch!

File: src/docker-compose.yml (hunk 1 above)
Comment (c-simpson) [reply src/docker-compose.yml:11]: That was an attempt to preserve the old pg13 install/data to make moving easier

**Comment by c-simpson** [comment]
Closes #571 
Right, @carrollsa  ?

**Comment by c-simpson** [comment]
Made the mistake of saying "Will cl*se  # 573"    in #597, so it did. Meant to merge in any case

**Pull request merged**